apps/vessel-history/features/terminology/terminology-iuu.tsx

```diff
@@ -9,15 +9,9 @@ export function TerminologyIuu(props: TerminologyIuuProps) {
     <div className={styles.container}>
       <Trans i18nKey="risk.iuuStatusDescription">
         Illegal, unreported and unregulated (IUU) listing status according to official RFMO IUU
-        vessel lists. There are two statuses:{' '}
-        <ol>
-          <li>vessel is currently listed: when a vessel is currently listed on an RFMO IUU list</li>
-          <li>
-            vessel is not currently listed: when a vessel is not currently listed on an RFMO IUU
-            list. This includes if a vessel was previously on an RFMO IUU list but has since been
-            removed.
-          </li>
-        </ol>
+        vessel lists. A vessel is defined as “present on an RFMO blacklist” if the vessel is
+        currently listed.” For more information, please see the{' '}
+        <a href="https://drive.google.com/drive/folders/1njcl_iDFiaLALTT-bK_wdlil7ldIntU8">FAQs</a>.
```

Comment:
I think we should store this url in the translations so that it can be localized in crowdin. WDYT?

Reply:
it is a folder, I asked her to know